bin/generate-offline-files.js

```diff
@@ -22,9 +22,9 @@ function processResults(results) {
     .reduce((current, next) => next.concat(current), [])
     .split("\n")
     .filter(item => item)
-    .map(item => item.replace(/^src\//, "/"))
+    .map(item => item.replace(/^src\//, "./"))
     .sort();
-  return JSON.stringify(paths.concat(["/"]), null, 2);
```

Comment:
We can't load the "base" any more this way, the index.html file now handles its own storage... see `addIndexToCache()` task below.